ssg/utils.py

```diff
@@ -179,9 +179,17 @@ def parse_name(product):
     return prod_tuple(_product, _product_version)
 
 
+def parse_platform(platform):
+    """
+    From a platform line, returns the set of platforms listed.
+    """
+
+    return set(map(lambda x: x.strip(), platform.split(',')))
+
+
```

Comment:
It has been transferred from the `build_yaml` module and renamed to reflect the scope of this new-old function, that is now used only to parse platforms.